project/AppDependencies.scala

```diff
@@ -0,0 +1,32 @@
+import play.core.PlayVersion
+import sbt._
+
+object AppDependencies {
+  private lazy val silencerVersion                   = "1.7.9"
+  private lazy val monocleVersion       = "1.7.3"
+  private lazy val bootstrapPlayVersion = "7.0.0"
+  private lazy val hmrcMongoPlayVersion = "0.68.0"
```

Comment:
if we are doing full verison bump please update this to 0.71.0 as well

Reply:
done